packages/components/Button/src/Button.tsx

```diff
@@ -25,6 +25,14 @@ export const Button = compose<IButtonType>({
     } = userProps;
     // attach the pressable state handlers
     const pressable = useAsPressable({ ...rest, onPress: onClick });
+    const onKeyDown = React.useCallback(
+      e => {
+        if (onClick && e.nativeEvent.key === 'Enter') {
+          onClick();
```

Comment:
If you have handled the event, you need to call `e.stopPropagation()` to stop it from bubbling.